app/src/main/res/values/strings.xml

```diff
@@ -1,10 +1,11 @@
 <resources>
-    <string name="common_ok">OK</string>
-    <string name="common_yes">Yes</string>
-    <string name="common_no">No</string>
-    <string name="common_cancel">Cancel</string>
-    <string name="common_delete">Delete</string>
-    <string name="common_remove">Remove</string>
+    <!-- Common strings, uncomment if you want to use them -->
+    <!--<string name="common_ok">OK</string>-->
+    <!--<string name="common_yes">Yes</string>-->
+    <!--<string name="common_no">No</string>-->
+    <!--<string name="common_cancel">Cancel</string>-->
+    <!--<string name="common_delete">Delete</string>-->
+    <!--<string name="common_remove">Remove</string>-->
 
```

Comment:
I agree with @shadow7 here.  I guess I've always been kind of uncomfortable with these - it's certainly not a given that every app will need all of these, and they're not used at all in the Skeleton at present.  Having strings for "yes"/"no"/"cancel" isn't rocket science and I think people can figure out how to do that themselves.  I'd rather they risk using different names for these than us having lint warnings that we go numb to ("yep there are a few warnings no big deal" can turn into "yeah 50 warnings is our baseline" before you know it).